corehq/apps/hqwebapp/utils/__init__.py

```diff
@@ -40,21 +40,6 @@ def sign(message):
     return signature
 
 
-def send_confirmation_email(invitation):
```

Comment:
I decided to delete this since (after the refactor) it was only used by the `Invitation` model and there was already a `send_activation_email` method inside of `Invitation`.

Reply:
on top of it, there is an unrelated `send_confirmation_email` view related to `CommCareUser`s, and that seemed like things could get confusing...